README.md

```diff
@@ -35,6 +35,41 @@ In addition to the minimum set above, a bindable service **SHOULD** provide:
 The key/value pairs insides this ConfigMap are:
 * A set of `metadata.<property>=<value>` - where `<property>` maps to one of the defined keys for this service, and `<value>` represents the description of the value.  For example, this is useful to define what format the `password` key is in, such as apiKey, basic auth password, token, etc.
 
+#### Making services discoverable
+
+The requirements above allow a bindable service to be discovered and processed by an implementation of this specification (such as the reference implementation, Service Binding Operator).  However, there is a desire for external tools to quickly query a list of bindable services without requiring them to process all of the scenarios outlined in this specification.  With this in mind, a further recommendation for bindable services is to add the following annotation to its bindable resource:
```

Comment:
I think the intent is not only to make it easy to query but also to mark some service providers as bindable, i.e. we would also want to not process service providers that we know won't be used for binding purposes so ideally, we would **only** inspect service providers that have that specific annotation… 
Unfortunately, unless we make this requirement a *MUST* and state that no provider will be processed unless it adds this annotation, spec implementations will still need to process all the operators…

Reply:
that's a valid point @metacosm.   In order to keep the "bar low" for adoption, we could perhaps say that the presence of any of the `servicebinding` annotations defaults to the equivalent of having `servicebinding/bindable: "true"`, from a processor's point of view.  

It's true that it puts more work on the processors, but I think that's ok since "we" (as implementors) are willing to do more to adopt the spec than the bindable services.  =)

In the cases where a bindable service's instance does not want to be bindable, it could then add the `servicebinding/bindable: "false"` annotation.

Reply:
I understand what you're saying but basically, if we don't enforce anything, implementations won't have a choice but to process everything because if it's not mandatory then its presence or absence cannot be asserted to be intent from the provider's side. So my question is: do we really need such an annotation if we cannot rely on it?

Reply:
Perhaps what we could say is that if implementations don't use this annotation they won't be listed in the results of "what's bindable".   If someone somehow "knows" about a bindable service that does not have this annotation, then it could use the `detectBindingResources` flag (https://github.com/application-stacks/service-binding-specification/issues/20) to bind to it.

@sbose78 - did you have any thoughts on this area?

Reply:
I agree with this stance which would more or less equate to making the annotation a MUST… 😁